scripts/run_astyle.sh

```diff
@@ -65,7 +65,7 @@ if [ ! -f "astyle_config.as" ]; then
     exit 2
 fi
 
-ASTYLE_FLAGS="--options=astyle_config.as ./\*.c ./\*.h"
+ASTYLE_FLAGS="--options=astyle_config.as --exclude=third_party/tomlc99 ./\*.c ./\*.h"
```

Comment:
Probably don't need the "/tomlc99" - we won't want to check any third-party libs.

Reply:
Indeed.